packages/commonwealth/server/models/chain.ts

```diff
@@ -66,6 +66,10 @@ export type ChainAttributes = {
   Users?: UserAttributes[] | UserAttributes['id'][];
   ChainObjectVersion?; // TODO
   Contract?: ContractInstance;
+
+  created_at?: Date;
+  updated_at?: Date;
+  deleted_at?: Date;
```

Comment:
No need for deleted_at -- we're not using paranoid mode.

Reply:
Sure, just raising that it was on the ticket in case @zakhap  had intended to start using paranoid mode or something? Will remove if not.

Reply:
We can add it later on if we have a use for it. For the time being, please remove it.